src/app/level/level-detail/level-detail.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="imglist">
+  <ol *ngFor="let topic of topics; let i = index" [class.selected]="topic.topicId === selectedId" (click)="onSelect(topic)">
+    <li><a [routerLink]="['/levels', level.levelId, '/questions', i+1]"><p>{{topic.name}}:</p> {{topic.description}}</a></li>
```

Comment:
This route is not defined, you can just remove the link and leave it for the next PR.